examples/crm/src/sales/SalesCreate.tsx

```diff
@@ -0,0 +1,19 @@
+import { Create, PasswordInput, SimpleForm } from 'react-admin';
+import { SalesForm } from './SalesForm';
+import { useIsAdmin } from './useIsAdmin';
+
+export function SalesCreate() {
+    const isAdmin = useIsAdmin();
+    if (!isAdmin) {
+        return null;
+    }
+
+    return (
+        <Create>
+            <SimpleForm>
+                <SalesForm />
```

Comment:
<!-- Notes are always non-blocking and simply highlight something the reader should take note of.  -->
**note (non-blocking):** 
When we create a new sale, its administrator status is not displayed in the list‏

Reply:
You should add an default value for adminstrator